bedrock/mozorg/redirects.py

```diff
@@ -479,15 +477,7 @@ def decider(request, **kwargs):
     redirect(r"^styleguide/identity/mozilla(.+)", "https://mozilla.design/mozilla/"),
     redirect(r"^styleguide(/.*)?", "https://mozilla.design/"),
     # Issue 8644, 8932
-    redirect(
-        r"^builders/?$",
-        "https://builders.mozilla.community/",
-        query={
-            "utm_source": "www.mozilla.org",
-            "utm_medium": "referral",
-            "utm_campaign": "builders-redirect",
-        },
-    ),
+    redirect(r"^builders/?$", "https://future.mozilla.org/builders/"),
```

Comment:
The utm tags got removed between the following redirects, so updated to just the final location…